src/napari/_qt/qt_resources/styles/02_custom.qss

```diff
@@ -195,6 +195,19 @@ QWidget[emphasized="true"] QDoubleSlider::handle:disabled {
     background: {{ primary }};
 }
 
+SliderLabel {
+  background-color: "white"; /*{{ darken(background, 15) }};*/
```

Comment:
Can you clarify this "white" and the commented out section?

Reply:
debug leftover